.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,14 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  misspell:
+    name: runner / codespell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: codespell
+        uses: plettich/action-codespell@v0.2
```

Comment:
So the action-codespell needs a repo.
Would you be willing to move this to privacyidea/ ?
I think we coudl in the long run end up with some of our own actions, so we might think about how to keep them in one place in a sensible way.

Reply:
Sure, You can fork it or we move it.
I've already created a pre-release in the GitHub Marketplace: https://github.com/marketplace/actions/run-codespell-with-reviewdog
Would You be willing to add the action to the Marketplace if we move it here?

Reply:
I think we start this way and check how we develop this in the future.